package.json

```diff
@@ -44,10 +44,11 @@
     "babel-runtime": "^6.0.0",
     "chai": "^3.5.0",
     "copy-webpack-plugin": "^4.0.1",
-    "cozy-bar": "beta",
+    "cozy-bar": "3.0.0-beta9",
     "cozy-client-js": "^0.1.0",
     "css-loader": "^0.26.0",
     "css-mqpacker": "^4.0.1",
+    "csswring": "^5.1.1",
```

Comment:
This one is ok and previously missing, see previous comment about embed postcss config :wink: